netbox/extras/urls.py

```diff
@@ -136,4 +136,8 @@
 
     # Markdown
     path('render/markdown/', views.RenderMarkdownView.as_view(), name="render_markdown"),
+
+    # Plugins
+    path('plugins/', views.PluginListView.as_view(), name='plugin_list'),
```

Comment:
This should go under `core` rather than `extras` (we just moved all the plugin resources out of `extras` in v3.7).